autoload/vista/executive/nvim_lsp.vim

```diff
@@ -55,9 +56,9 @@ function! s:RunAsync() abort
 
         if err then print(tostring(err)) return end
         if not result then return end
-        vim.g.vista_executive_nvim_lsp_fetching = false
         data = vim.fn['vista#renderer#LSPPreprocess'](result)
         vim.fn['vista#executive#nvim_lsp#SetData'](data)
+        vim.g.vista_executive_nvim_lsp_fetching = false
```

Comment:
Is this change really necessary?

Reply:
If not, there still a chance that `Run` stop fetching but `vista#executive#nvim_lsp#SetData` is not call and `s:data` is not overwrite by new data ?

Reply:
Right, `s:Run` needs to block until `s:data` is updated.